backend/app/services/llm_judge.py

```
@@ -4,151 +4,77 @@
-        }
-        with OUTPUT_FILE.open("w", encoding="utf-8") as f:
-            json.dump(results, f, ensure_ascii=False, indent=2)
+        raw_text = response.content if hasattr(response, "content") else str(response)
+        clean_text = raw_text.strip().lstrip("```json").rstrip("```").strip()
 
-        return results
+        try:
+            result = json.loads(clean_text)
+            expected_keys = {"usefulness", "groundedness", "efficiency"}
+            if not expected_keys.issubset(result.keys()):
+                raise ValueError("В ответе LLM отсутствуют обязательные ключи")
```

Ruff (0.15.12)

[warning] String contains ambiguous `В` (CYRILLIC CAPITAL LETTER VE). Did you mean `B` (LATIN CAPITAL LETTER B)?

(RUF001)